src/main/json2xlsx.js

```diff
@@ -22,31 +24,33 @@ const run = async (args) => {
   const builder = new ccdUtils.SpreadsheetBuilder(sourceXlsx);
   await builder.loadAsync();
 
-  const excludedFilenamePatterns = args.excludedFilenamePatterns ? stringUtils.split(args.excludedFilenamePatterns) : [];
-  const files = fileUtils.listFilesInDirectory(args.sheetsDir, excludedFilenamePatterns);
+  const excludedFilenamePatterns = args.excludedFilenamePatterns ?
+    stringUtils.split(args.excludedFilenamePatterns) : [];
 
-  for (const file of files) {
-    const readSheetData = async (file) => {
-      const readJsonFile = (relativeFilePath) => {
-        return fileUtils.readJson(path.join(args.sheetsDir, relativeFilePath), Substitutor.injectEnvironmentVariables);
-      };
+  const paths = await fileUtils.getJsonFilePaths(args.sheetsDir, excludedFilenamePatterns);
+  const fileMap = await sheetUtils.groupToSheets(paths);
 
-      if (file.isDirectory()) {
+  for (const file in fileMap) {
```

Comment:
Please name it appropriately. Name `file` suggests me that it is some sort of file handler that defined rich set of file operations while it is just file name (string).

IMO in JS since there is no type information names should more clearly describe what they represent. In typed languages type would more clearly express that it is a string for example.

Reply:
`sheetName` maybe with the map being called `sheetFileMap`?